master/getting-started/kubernetes/installation/config-options.md

```diff
@@ -149,6 +149,26 @@ To use these manifests with a TLS-enabled etcd cluster you must do the following
 Depending on your cluster's authorization mode, you'll want to back these
 service accounts with the necessary permissions.
 
+### Configuring service advertisement
+
+{{site.prodname}} supports [advertising Kubernetes services over
+BGP]({{site.baseurl}}/{{page.version}}/usage/service-advertisement),
+so that service cluster IPs are routable from outside the cluster.  To
+enable this, add an `ADVERTISE_CLUSTER_IPS` variable setting to the
+environment for {{site.nodecontainer}} in the `calico.yaml` manifest,
+with value equal to the cluster IP range for your Kubernetes cluster;
+for example:
+
+```yaml
+          env:
+            [...]
+            - name: ADVERTISE_CLUSTER_IPS
```

Comment:
Also, looks like the code currently supports `CALICO_ADVERTISE_CLUSTER_IPS`.

So, either need to make the code support both or update this value here. WDYT?

Reply:
I was going by the gdoc; but if you're happy that `CALICO_ADVERTISE_CLUSTER_IPS` is more the Right Thing, I'll just update the doc to say that.

(Unfortunately we seem to have got ourselves into a mess here, on the point of whether or not we prefix with CALICO; see https://docs.projectcalico.org/master/reference/node/configuration.)

Reply:
Sigh, yeah...

I'm not sure. It seems like the way to align all of those inconsistent variables would be to support both a `CALICO_` prefixed and not prefixed variants. 

I think this is what we do for other bits of config as well, for example ETCD_ENDPOINTS and CALICO_ETCD_ENDPOINTS, where the CALICO_ prefixed version takes precedence. 

We get this for free by using `envconfig`.